### PR TITLE
feat: add list of partners component

### PR DESCRIPTION
**Pull request by idzeravianka**

### CODE REVIEW

**Comment by idzeravianka** [comment]
[Landing Page (Partners Page)](https://trello.com/c/dl34uS1q/13-landing-page-partners-page)

File: js_machine_front/src/routes/Partners.tsx [hunk 1]
```diff
@@ -13,7 +15,10 @@ const sectionStyle = {
 export const Partners: React.FC = memo(() => {
   return (
     <div style={ sectionStyle }>
-      <div className="body" />
+      <div className="body">
```
Comment (vharadkou) [inline js_machine_front/src/routes/Partners.tsx]: body classname? maybe this should be partners?

**Review by vharadkou — CHANGES_REQUESTED** [review]

File: js_machine_front/src/components/PartnersLogo.tsx [hunk 2]
```diff
@@ -0,0 +1,18 @@
+import React from 'react';
+import 'styles/partnersLogo.css';
+
+import EpamLogo from 'images/epam-icon.png';
+import DevByLogo from 'images/dev-by-icon.png';
+import StreamLineLogo from 'images/streamline-icon.png';
+
+const PartnersLogo: React.FC = () => {
```
Comment (vharadkou) [inline js_machine_front/src/components/PartnersLogo.tsx]: use memo

File: js_machine_front/src/components/PartnersLogo.tsx [hunk 3]
```diff
@@ -0,0 +1,18 @@
...
+    return (
+        <div className="partners-logo">
+            <img src={EpamLogo} className="partners-logo__img" alt="Epam logo"/>
+            <img src={DevByLogo} className="partners-logo__img" alt="Dev.by logo"/>
+            <img src={StreamLineLogo} className="partners-logo__img" alt="Streamline logo"/>
+        </div>
+    );
+};
+
+export default PartnersLogo;
```
Comment (vharadkou) [inline js_machine_front/src/components/PartnersLogo.tsx]: don't use default export, always export const at the beginning

**Review by vharadkou — CHANGES_REQUESTED** [review]

**Review by vharadkou — APPROVED** [review]

**Pull request merged**